beacon_constants.go

```diff
@@ -40,7 +40,9 @@ const blockHeaderMerkleSubtreeNumLayers = uint64(3)
 const blockBodyMerkleSubtreeNumLayers = uint64(4)
 
 // Number of layers for the merkelization of the Execution Payload
-const executionPayloadMerkleSubtreeNumLayers = uint64(4)
+const executionPayloadMerkleSubtreeNumLayers = uint64(5)
```

Comment:
please name things with Deneb and Capella for clarity 🙏 will make it easier in the future 😂